### ISSUE TITLE
Composer Segmentation fault

### ISSUE THREAD

**Issue by bigtunacan** [rlerdorf/php7dev#63]
Setup the fresh Vagrant VM, created host file entry, visited http://php7dev; so far so good.

Then I tried testing composer; just did `composer require twig/twig:~1.8` as a quick test and I just get "Segmentation fault" as the output.  Tried using sudo; same thing.  Then tried just `composer` by itself; same thing "Segmentation fault".

I can provide more info if it helps, just not sure where to start.

**Comment by rlerdorf** [comment]
Which PHP version are you trying?

Does it work under 7.0? Type `newphp 70` to switch. If you are using 7.1, the version on there is a bit early. Build the latest with `makephp 71`

**Comment by bigtunacan** [comment]
@rlerdorf, I was using 7.1. I had already tried rebuilding latest with makephp 71, but that did not fix the issue.

I switched over to 7.0 as you've suggested; that fixes it and I'm fine with 7.0 for what I'm currently working on.  Thanks for your help!

**Comment by kvdnberg** [comment]
I'm still experiencing this problem with PHP 7.1. I just installed the box fresh, ran `makephp 71`, ran composer without parameters (I was just checking if it was there, I need this box with composer for a tutorial at DPC) and got `Segmentation fault`.

I tried `sudo apt-get update` && `sudo apt-get upgrade` but it made no difference.

Is there any way to get this combination working (PHP 7.1 & composer)?
